src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -157,76 +157,76 @@ bool Deconvolution::isSupportedOperation(const std::shared_ptr<const ngraph::Nod
 
-            outputPadding = groupConvBackprop->get_output_padding();
+        groupNum = weightDims[0];
+        IC = groupNum * weightDims[1];
+        OC = groupNum * weightDims[2];
+        expectedBiasDims = {OC * groupNum};
+        withGroups = groupNum > 1;
+        isDW = withGroups && groupNum == OC && groupNum == IC;
 
-            autoPad = one_of(groupConvBackprop->get_auto_pad(), ov::op::PadType::SAME_LOWER, ov::op::PadType::SAME_UPPER);
+        for (int i = 0; i < groupConvBackprop->get_strides().size(); i++) {
+            stride.push_back(static_cast<ptrdiff_t>(groupConvBackprop->get_strides()[i]));
         }
-        for (int i = 0; i < dilation.size(); i++) {
-            kernel.push_back(weightDims[withGroups + 2 + i]);
+        for (int i = 0; i < groupConvBackprop->get_dilations().size(); i++) {
+            dilation.push_back(static_cast<ptrdiff_t>(groupConvBackprop->get_dilations()[i]) - 1);
         }
+        paddingL = groupConvBackprop->get_pads_begin();
+        paddingR = groupConvBackprop->get_pads_end();
 
-        externOutShape = inputShapes.size() == 3;
-        biasPort = externOutShape ? 3 : 2;
-        if (externOutShape && isDynamicNode()) {
-            bool isConstOutShape = ngraph::is_type<ov::op::v0::Constant>(op->get_input_node_shared_ptr(2));
-            if (isConstOutShape) {
-                lastOutputSpatialDims = ov::as_type<ov::op::v0::Constant>(op->get_input_node_ptr(2))->cast_vector<int32_t>();
-            }
-            const auto spDimsNum = getInputShapeAtPort(0).getRank() - 2;
-            if (getInputShapeAtPort(2).getStaticDims()[0] != spDimsNum || (isConstOutShape && lastOutputSpatialDims.size() != spDimsNum)) {
-                IE_THROW() << "'output_shape' input has incorrect number of elements. Expected = " << spDimsNum;
-            }
+        outputPadding = groupConvBackprop->get_output_padding();
+
+        autoPad = one_of(groupConvBackprop->get_auto_pad(), ov::op::PadType::SAME_LOWER, ov::op::PadType::SAME_UPPER);
+    }
+    for (int i = 0; i < dilation.size(); i++) {
+        kernel.push_back(weightDims[withGroups + 2 + i]);
+    }
+
+    externOutShape = inputShapes.size() == 3;
+    biasPort = externOutShape ? 3 : 2;
+    if (externOutShape && isDynamicNode()) {
+        bool isConstOutShape = ngraph::is_type<ov::op::v0::Constant>(op->get_input_node_shared_ptr(2));
+        if (isConstOutShape) {
+            lastOutputSpatialDims = ov::as_type<ov::op::v0::Constant>(op->get_input_node_ptr(2))->cast_vector<int32_t>();
+        }
+        const auto spDimsNum = getInputShapeAtPort(0).getRank() - 2;
+        if (getInputShapeAtPort(2).getStaticDims()[0] != spDimsNum || (isConstOutShape && lastOutputSpatialDims.size() != spDimsNum)) {
+            IE_THROW() << "'output_shape' input has incorrect number of elements. Expected = " << spDimsNum;
         }
-    } else {
-        IE_THROW(NotImplemented) << errorMessage;
     }
 
```

Comment:
The compare has difficulty aligning above code snippets,  essentially you just replaced `biasesDims` with `expectedBiasDims` and switched coding style to `Guard clause`, just for other reviewers information